deeplearning4j-keras/src/main/java/org/deeplearning4j/keras/DeepLearning4jEntryPoint.java

```diff
@@ -25,14 +23,15 @@ public void fit(EntryPointFitParameters entryPointFitParameters) throws Exceptio
         try {
             MultiLayerNetwork multiLayerNetwork = neuralNetworkReader.readNeuralNetwork(entryPointFitParameters);
 
```

Comment:
What about ComputationGraph and sequential? This should only be a few lines of code difference anyways.